profanity_word_phrase_filter/README.md

```diff
@@ -1,19 +1,20 @@
 ## Custom profanity filter
 
-Most online platforms contain inappropriate language. Parent's want to be certain
+Most online platforms contain inappropriate language. Parents want to be certain
 that Story squad (aka scribble stadium) is offering a safe environment for their
-kiddos. In compliance to the children's Online Protection rule (COPPA)
+kiddos in compliance with the Children's Online Privacy Protection Rule (COPPA).
```

Comment:
COPPA pertains only to privacy, and has nothing to do with inappropriate language. These two ideas need to be separated into two separate paragraphs (one for COPPA and privacy, and one for content moderation and inappropriate language)